Doc/library/stdtypes.rst

```diff
@@ -474,6 +474,24 @@ class`. In addition, it provides a few more methods:
 
     .. versionadded:: 3.1
 
+.. method:: int.bit_count()
+
+    Return the number of ones in the binary representation of the integer,
+    exluding the sign bit::
+
+        >>> n = -19
+        >>> bin(n)
+        '-0b10011'
+        >>> n.bit_count()
+        3
+
+    Equivalent to::
+
+        def bit_count(self):
+            return bin(self).count("1")
+
+    .. versionadded:: 3.7
```

Comment:
Needs updating if this is going into 3.8 (and also if it's going into 3.9, of course).